### PR TITLE
tests: remove explicit local storage tests

### PR DESCRIPTION
**Pull request by aditya-mitra**
## Summary

File browser should not interact with the storage provider. 
- removed separate tests for each storage provider. Both storage providers were testing the same cases. 
- fix parallel mocha tests not working for storage provider
- fix getSignedUrl in local.storage
- use `PutCommand` in s3 putObject (and not local s3 mode) 

## Checklist
- [ ] If this PR is still a WIP, convert to a draft
- [ ] When this PR is ready, mark it as "Ready for review"
- [ ] [ensure all checks pass](https://github.com/etherealengine/etherealengine/wiki/Testing-&-Contributing)
- [ ] Changes have been manually QA'd
- [ ] Changes reviewed by at least 2 approved reviewer

## QA Steps

_List any additional steps required to QA the changes of this PR, as well as any supplemental images or videos._

### CODE REVIEW

File: packages/server-core/package.json [hunk 1]
```diff
@@ -97,7 +97,6 @@
     "lodash": "4.17.21",
     "meshoptimizer": "^0.18.1",
     "mime-types": "^2.1.35",
-    "minio": "7.1.3",
```
Comment (hanzlamateen) [inline packages/server-core/package.json:100]: We dont need to remove this. Because there are some scripts that are using storage provider and if we remove it from here then it will create issues for those scripts.

**Review by hanzlamateen — CHANGES_REQUESTED** [review]

**Review by hanzlamateen — APPROVED** [review]
Look good to me. But maybe @HexaField can fix a 2nd look of changes in the tests.

File: packages/server-core/tests/storageprovider/storageprovider.test.ts [hunk 2]
```diff
@@ -56,133 +56,135 @@ describe('storageprovider', () => {
...
-          provider.putObject({
-            Body: data,
-            Key: fileKey,
-            ContentType: getContentType(fileKey)
-          })
-        )
-      }
-      await Promise.all(promises)
-    })
+      it(`should put over 100 objects in ${providerType.name}`, async function () {
```
Comment (HexaField) [inline packages/server-core/tests/storageprovider/storageprovider.test.ts:163]: why the change from 1000 to 100?

**Review by HexaField — APPROVED** [review]
